### ISSUE TITLE
Edit employee module - tabs cannot be reachable via the keyboard

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8586]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

The tabs in the edit employee modal on the company detail page cannot be reachable via the keyboard.
There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
https://docs.deque.com/issue-help/1.0.0/en/keyboard-inaccessible

![Image](https://github.com/user-attachments/assets/e1d96a90-ed29-4a9f-86b6-9b4139c2b677)

### Expected behaviour

The tabs in the edit employee module on the company detail page should be reachable via the keyboard.

### How to reproduce

Try to reach the tabs in the role details on the company detail page via the keyboard.
